end_to_end/2-lineage-train-assess-bias-tune-registry-e2e.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>both data artifact and model artifact can be found; I don't understand why the code artifact cannot be found; I went the s3 location and verified that <code>sourcedir.tar.gz</code>  is there. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Curious: Are you running in SageMaker Studio?</p>

Reply:
<p>I have re-run and it does not give me that error</p>

Reply:
That's a good point. I get that error when running on an EC2, but didn't get the error when running on an SM instance. I haven't tried  to run in Studio yet.